Testnet/StandardToken/StandardToken/StandardToken.cs

```diff
@@ -120,7 +117,7 @@ public bool TransferFrom(Address from, Address to, ulong amount)
     }
 
     /// <inheritdoc />
-    public bool Approve(Address spender, ulong currentAmount, ulong amount)
+    public bool Approve(Address spender, UInt256 currentAmount, UInt256 amount)
```

Comment:
Maybe @rowandh might remember. 

Is there a specific reason for requiring the sender to send their `currentAmount` for this spender? The only reason I can think is to force the user to check their own allowance first. But they aren't going to do that, its just extra work to get the users allowance to use as the currentAmount property that average users never even know is happening. 

interested in possibly ripping this piece out as I don't see much value.

Thoughts? @quantumagi @YakupIpek